integration/rocket-chat/main.js

```diff
@@ -579,23 +595,29 @@ function SmartiWidget(element, _options) {
                  *
                  * @param {Object} data
                  * @param {Object} data.response
-                 * @param {Number} data.response.numFound
-                 *
+                 * @param {Object} data.items
                  */
                 success: (data) => {
```

Comment:
Will never get here, because google does not accept XHR-requests.
Google custom search API offers a `callback` function but only for GET requests send by the client:
```
<script src="https://www.googleapis.com/customsearch/v1?key=YOUR-KEY&cx=017576662512468239146:omuauf_lfve&q=cars&callback=hndlr">
```
See: https://developers.google.com/custom-search/v1/using_rest?refresh=1#rest_from_javascript